lecture_3/03_project/calculator/cmd/config/config.go

```diff
@@ -0,0 +1,45 @@
+package config
+
+import "github.com/kelseyhightower/envconfig"
+
+// Cfg is the single instance of configuration that gets automatically populated from the
+// environment variables once the  module loads.
+var Cfg Config
+
+// Config contains all the configuration needed for service to work.
+type Config struct {
+	Rabbit         rabbitConfig `split_words:"true"`
+	SqliteDatabase string       `split_words:"true"`
+}
+
+type rabbitConfig struct {
+	ConsumerEventUpdateQueue    string `split_words:"true" required:"true"`
+	ConsumerBetQueue            string `split_words:"true" required:"true"`
+	ConsumerEventUpdateName     string `split_words:"true" default:"controllerreceived"`
+	ConsumerBetName             string `split_words:"true" default:"controllercalculated"`
```

Comment:
These default values are from Controller, I've used some made-up naming style which is a hybrid of `<service_name><topic_name>`, but reusing the same naming in multiple services is not good. RabbitMQ will identify those consumers as the same consumer, which is not desired - e.g. if they were consuming the same queue, then every message would have been read only by one service.